api/schema-directives.graphql

```diff
@@ -202,7 +202,7 @@ directive @clearCache(
   Name of the field to clear.
   """
   field: String
-) on FIELD_DEFINITION
+) repeatable on FIELD_DEFINITION
```

Comment:
Any reason why the schema directives needed to change for this branch?

Reply:
I have no idea what this was all about 🤷